DBMS/CommandJoin/question1/Create_Script.sql

```diff
@@ -0,0 +1,68 @@
+drop database StoreFront;
```

Comment:
The convention is to use small letters and underscores for separating words within variables names (db name, table name, etc)